.github/README.md

```diff
@@ -1,50 +1,50 @@
+## Documentation
 
-To view more examples, please visit [https://umbraco.com/case-studies-testimonials/](https://umbraco.com/case-studies-testimonials/)
+The full documentation for Umbraco CMS can be found [here](https://our.umbraco.com/documentation/). The source for the Umbraco docs is hosted on a separate repo - https://github.com/umbraco/UmbracoDocs - where we'd also love to have your helping hand!
 
-## Why Open Source?
-As an Open Source platform, Umbraco is more than just a CMS. We are transparent with our roadmap for future versions, our incremental sprint planning notes are publicly accessible, and community contributions and packages are available for all to use.
+## Contributing
 
-## Trying out Umbraco CMS
-
-[Umbraco Cloud](https://umbraco.com/cloud) is the easiest and fastest way to use Umbraco yet, with full support for all your custom .NET code and integrations. You're up and running in less than a minute, and your life will be made easier with automated upgrades and a built-in deployment engine. We offer a free 14-day trial, no credit card needed.
-
-If you want to DIY, you can [download Umbraco](https://our.umbraco.com/download) either as a ZIP file or via NuGet. It's the same version of Umbraco CMS that powers Umbraco Cloud, but you'll need to find a place to host it yourself, and handling deployments and upgrades will be all up to you.
-
-## Community
+Umbraco is contribution-focused and community-driven. If you want to contribute back to Umbraco, please check out our [guide to contributing](CONTRIBUTING.md).
 
-Our friendly community is available 24/7 at the community hub we call ["Our Umbraco"](https://our.umbraco.com). Our Umbraco features forums for questions and answers, documentation, downloadable plugins for Umbraco, and a rich collection of community resources.
+## Join the Umbraco community
 
-## Contribute to Umbraco
+Our friendly community is available 24/7 at the community hub we call ["Our Umbraco"](https://our.umbraco.com/). Our Umbraco features forums for questions and answers, documentation, downloadable plugins for Umbraco, and a rich collection of community resources.
 
-Umbraco is contribution-focused and community-driven. If you want to contribute back to Umbraco, please check out our [guide to contributing](CONTRIBUTING.md).
+Besides "Our", we all support each other also via:
 
-## Found a bug?
+- Twitter: [Umbraco HQ](https://twitter.com/umbraco), [Release Updates](https://twitter.com/umbracoproject), [#umbraco](https://twitter.com/hashtag/umbraco)
+- Gitter: https://gitter.im/umbraco/Umbraco-CMS
+- Slack: http://umbracians.chat/
```

Comment:
Hi @mzajkowski,

The URL for Slack is no longer working, and I don't think it has been working for some months. The URL just results in a 403 on Azure. Could you remove the link? From what I've seen on Slack, there isn't a new URL to be used instead.